pyproject.toml

```
@@ -97,11 +76,11 @@ import_heading_firstparty = "Local Imports"
 
 # Pylint settings
 [tool.pylint.basic]
-good-names = ["df", "fn", "ca", "tp", "to", "x", "x1", "x2", "i", "f", "q", "mu"]
+good-names = ["df", "fn", "ca", "tp", "to", "x", "x1", "x2", "i", "f", "q"]
 extension-pkg-allow-list = ["pydantic"]
 
 [tool.pylint.main]
-ignore = ["__init__.py", "_version.py"]
+ignore = ["__init__.py", "_version.py", "versioneer.py"]
```

Shouldn't be needed anymore as we don't use versioneer
```suggestion
ignore = ["__init__.py", "_version.py"]
```